wayland-commons/src/debug.rs

```diff
@@ -0,0 +1,71 @@
+//! Debugging helpers to handle `WAYLAND_DEBUG` env variable.
+
+use std::time::{SystemTime, UNIX_EPOCH};
+
+use crate::wire::Argument;
+
+/// Print the dispatched event to stderr in a following format:
+///
+/// [timestamp] <- interface@id.msg_name(args)
+pub fn print_dispatched_event(interface: &str, id: u32, msg_name: &str, args: &[Argument]) {
+    // Add timestamp to output.
+    print_timestamp();
+
+    eprint!(" <- {}@{}.{}", interface, id, msg_name);
+
+    print_args(args);
+
+    // Add a new line.
+    eprintln!();
+}
+
+/// Print the send event to stderr in a following format:
+///
+/// [timestamp] -> interface@id.msg_name(args)
+///
+/// If `is_alive` is `false` the `[ZOMBIE]` is added after `id`.
+pub fn print_send_event(
+    interface: &str,
+    id: u32,
+    is_alive: bool,
+    msg_name: &str,
+    args: &[Argument],
+) {
+    // Add timestamp to output.
+    print_timestamp();
+
+    eprint!(" -> {}@{}{}.{}", interface, id, if is_alive { "" } else { "[ZOMBIE]" }, msg_name);
+
+    print_args(args);
+
+    // Add a new line.
+    eprintln!();
+}
+
+/// Print arguments without opening/closing bracket.
```

Comment:
Given the code of this function, I think the comment should be "with" and not "without"